src/agent.ts

```diff
@@ -638,6 +724,10 @@ export class Agent {
 
     await this.interactionManager.start(token)
 
+    this.logger.info(
+      `RPC encryption request received. Nonce: '${token.nonce}'.`,
```

Comment:
All these tokens are not received, but created by the agent. Also since there is already a channel for interactions, we could log these in there, since all these tokens will be processed there as well.

Reply:
Agree. Will be done in separate PR (decoupled from service management feature).